rpc_util_test.go

```diff
@@ -170,3 +171,41 @@ func TestBackoff(t *testing.T) {
 		}
 	}
 }
+
+// bmEncode benchmarks encoding a simple Protocol Buffer message containing
```

Comment:
Done.  This is used to scaffold strictly the the `Buffer` message type for now.  I had included 'simple' originally due to just using an array of zero-ed bytes as opposed to another predefined byte payload or even randomly-generated.